taskgraph.json

```diff
@@ -30,12 +30,12 @@
           "image": "quay.io/mozilla/gaia-taskenv:0.8.9",
           "command": [
             "entrypoint",
-            "echo 'what' && node tests/taskcluster/bin/graph > /graph.json && node tests/taskcluster/bin/raptor_graph > /raptor_graph.json && cat /graph.json && cat /raptor_graph.json"
+            "echo 'what' && node tests/taskcluster/bin/graph > /graph.json && node tests/taskcluster/bin/raptor_decision > /raptor_decision.json && cat /graph.json && cat /raptor_decision.json"
```

Comment:
Is there anything specific in bin/raptor_decision that it needs to be separate? Would it be possible to just remove this entirely, add a raptor_decision yml file in gaia/tests/taskcluster/tasks that is just added as part of the graph.json?